fs/fspath/path_test.go

```diff
@@ -177,8 +179,12 @@ func TestParse(t *testing.T) {
 				Path:         "/path/to/file",
 			},
 		}, {
-			in:      "rem.ote:/path/to/file",
-			wantErr: errInvalidCharacters,
+			in: "rem.ote:/path/to/file",
```

Comment:
You should change this test so it has a different invalid character - same with the ones below and maybe insert a `.` or two in currently passing tests.

Reply:
Yes, of course. Will do.